src/kaleidoscope/driver/keyscanner/NoOpKeyScanner.h

```diff
@@ -0,0 +1,36 @@
+/* -*- mode: c++ -*-
+ * NoOpKeyScanner -- Dummy keyscanner component
```

Comment:
Can this grow some comments about when one might use it?

Reply:
On any device where you don't have a key scanner!

In reality, this is just a thin wrapper around `BaseKeyscanner`, so that the base device description can use `NoOpKeyScanner` instead of `BaseKeyscanner`. That's all the reason for its existence.

I'll comment it up, along with the other, similar "alias" classes.